include/Editor.h

```diff
@@ -62,6 +62,8 @@ private slots:
 	/// Called by pressing shift+space. Toggles pause state.
 	void togglePause();
 
+	void toggleMaximise();
```

Comment:
Dang it, missed this one earlier. Once again, I'd prefer if this was consistent with the existing isMaximized function's name.

Reply:
Ughhh I'll fix it in a commit in sometime

Reply:
Just updated, now fixed.